source/HL_spi.c

```diff
@@ -1880,7 +1880,210 @@ void mibspi1HighLevelInterrupt(void)
 
 
 
+/* USER CODE BEGIN (50) */

```

Comment:
Did you change anything in this file?

Reply:
I enabled the interrupts for SPI3 in halocgen, and it generated the following code for me